synapse/lib/lmdbslab.py

```diff
@@ -1623,7 +1623,19 @@ def forcecommit(self):
         self.commitstats.append((starttime, xactopslen, delta))
 
         if self.WARN_COMMIT_TIME_MS and delta > self.WARN_COMMIT_TIME_MS:
-            logger.warning(f'Commit with {xactopslen} items in {self!r} took {delta} ms.')
+            logger.warning(f'Commit with {xactopslen} items in {self!r} took {delta} ms - performance may be degraded.')
```

Comment:
Can we add the `xactopslen` and `delta` values to the extras so that information is available in structlog output?

Reply:
Updated